instrumentation/netty-4.1.16/src/main/java/com/agent/instrumentation/netty4116/Http2RequestHeaderWrapper.java

```diff
@@ -28,12 +29,14 @@
 
 public class Http2RequestHeaderWrapper extends ExtendedRequest {
     private static final Pattern URL_REPLACEMENT_PATTERN = Pattern.compile("(?i)%(?![\\da-f]{2})");
+    private static final String GRPCHEADERSUTIL = "io.grpc.netty.GrpcHttp2HeadersUtils.GrpcHttp2RequestHeaders";
```

Comment:
Can this be changed to `GRPC_HEADER_UTIL_CLASS`?

Reply:
done